sql/core/src/main/scala/org/apache/spark/sql/execution/SparkSqlParser.scala

```diff
@@ -877,6 +877,35 @@ class SparkSqlAstBuilder(conf: SQLConf) extends AstBuilder {
   }
 
   /**
+   * Create a [[AlterTableChangeColumnsCommand]] command.
+   *
+   * For example:
+   * {{{
+   *   ALTER TABLE table [PARTITION partition_spec]
+   *   CHANGE [COLUMN] `col` `col` dataType [COMMENT "comment"] [FIRST | AFTER `otherCol`]
+   *   [, `col2` `col2` dataType [COMMENT "comment"] [FIRST | AFTER `otherCol`], ...]
```

Comment:
What is the reason do we allow users to modify multiple columns in the same DDL? This is different from what Hive supports. Should we do it? cc @cloud-fan 

In addition, based on the existing syntax, if we really want to support multiple columns, we should change the keywords to `CHANGE COLUMNS`.